vue-components/package.json

```
@@ -2,25 +2,51 @@
   "name": "@wmde/wikit-vue-components",
   "version": "0.0.0",
   "description": "The component implementation of wikit in vue",
-  "author": "The Wikidata team",
-  "homepage": "https://github.com/wmde/wikit/tree/master/vue-components#readme",
-  "license": "BSD-3-Clause",
+  "author": {
+    "name": "The Wikidata team"
+  },
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --target lib --formats commonjs --name wikit-vue-components src/main.ts --report --report-json",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint",
+    "test": "echo \"wikit vue components tests\" && exit 0"
```

This is again a matter of coordinating this between the patch here and https://phabricator.wikimedia.org/T257817 but if we add the basic test here, I think it should also be run to prove it is functional.

Maybe
```suggestion
    "test": "npm run lint && npm run test:unit"
```
for now.